frontstage/templates/surveys/surveys-add.html

```diff
@@ -1,17 +1,40 @@
 
 {% extends "layouts/_block_content.html" %}
+{% from "components/breadcrumbs/_macro.njk" import onsBreadcrumbs %}
 {% from "components/button/_macro.njk" import onsButton %}
+{% from "components/details/_macro.njk" import onsDetails %}
 {% from "components/input/_macro.njk" import onsInput %}
-
+{% from "components/panel/_macro.njk" import onsPanel %}
 {% set errorType = data['error']['type'] %}
+{% set panelVariant = "branded" %}
+{% set panelText = "" %}
+
+{% set page_title = "Add a new survey" %}
+
+{% set breadcrumbsData = [
+    {
+        "text": "Surveys",
+        "url": "/surveys/todo",
+        "id": "b-item-1"
+    }
+] %}
 
-{% set page_title = "Add a survey" %}
+{% block breadcrumbs %}
```

Comment:
The style inherited from the design system for the breadcrumbs that there is padding of 8px around the link text. Karen is happy for this to replace the "ons-u-mb-s" (18px) margin.